samples/display/lvgl/boards/mimxrt1050_evk.conf

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2020 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_I2C=y
```

Comment:
This board config files are a maintenance overhead sometimes. Why not enable it in [soc/arm/nxp_imx/rt/Kconfig.defconfig.series](https://github.com/zephyrproject-rtos/zephyr/blob/master/soc/arm/nxp_imx/rt/Kconfig.defconfig.series#L32) ?

Reply:
I didn't implement it originally because I was worried about complex Kconfig dependencies, but given your feedback I went ahead and did it now. Please have a look at the updated version.